nexus/README.md

```diff
@@ -24,16 +24,14 @@ This sample shows how to use Temporal for authoring a Nexus service and call it
    site](https://learn.temporal.io/getting_started/go/dev_environment/#set-up-a-local-temporal-service-for-development-with-temporal-cli)
    to install Temporal CLI.
 
-> NOTE: Required version is at least v1.1.0.
```

Comment:
Do we want to recommend `v1.3.0` (I am assuming that will be the latest release and include all the GA changes)

Reply:
I was debating that but didn't want to be too strict. Rethinking that I think it's probably the best to recommend the latest and greatest.

Reply:
I think if we merge https://github.com/temporalio/sdk-go/pull/1833 we should only be recommending a server release that understands this error shouldn't be retryable

Reply:
Updated the recommendation.